### ISSUE TITLE
cuelang.org: frontend: documented methods to resize inline images don't appear to work

### ISSUE THREAD

**Issue by jpluscplusm** [cue-lang/cue#2810]
### What did you do?

I attempted to create an alpha.cuelang.org page with an inline image that has a reduced visual size:

- Page: https://cl-1176774-1--cue-cls.netlify.app/docs/howto/resize-an-image/
- CL: https://review.gerrithub.io/c/cue-lang/cuelang.org/+/1176774

### What did you expect to see?

At the very least I expected to see the `{{< figure >}}` shortcode example work, as per https://alpha.cuelang.org/examples/basic/images/#figure-shortcode.

### What did you see instead?

I only found one mechanism that allows me to include a down-scaled inline image: raw HTML with a style attribute:

```
<img src="/blog/example/cow.jpg" style="width:75px;" />
``` 

We should either [document this](https://alpha.cuelang.org/examples/basic/images/), if it's the only way to resize an inline image, or make the documented methods work.

**Comment by myitcv** [comment]
@jpluscplusm - please can you confirm if this is still an issue and if so that the description above is accurate?

**Comment by jpluscplusm** [comment]
The issue still stands, but I'm not sure how much of an issue it is.

We ran into this when trying to format the section that's now visible at https://cuelang.org/docs/reference/modules/#minimal-version-selection.

We had an image which was too large, and wanted to reduce its size. We had been planning to place it as follows:

```
prose
more prose
more prose

[ reduced size image ]

some more prose
...
```

... but weren't able to do this, as described in this issue. We ended up solving this issue by using the right-aligned image that's visible on that page today, via the `figure` shortcode. I think it looks fine, and that this issue wasn't a blocker on being able to produce that page. 

I think the question is: how *often* will we need an image to be resized on the page, where it *cannot* be presented with text to its left or right? That's the use case that this issue captures, and I'm unsure how significant it is. I don't think it's currently a problem, but it's conceivable that we'll try to create a piece of content that has a harder requirement on such an impossible layout.
